pkg/util/proxy.go

```diff
@@ -158,13 +158,14 @@ func (p *ProxyConfig) IsEnabled() bool {
 }
 
 // ValidateProxyURL validates that the specified proxyURL is valid
-func ValidateProxyURL(proxyUrl string, scheme string) error {
+func ValidateProxyURL(proxyUrl string, defaultScheme string) error {
 	if proxyUrl == "" {
 		return nil
 	}
 
-	if !strings.HasPrefix(proxyUrl, scheme) {
-		proxyUrl = fmt.Sprintf("%s://%s", scheme, proxyUrl)
+	if !strings.HasPrefix(proxyUrl, "http://") &&
+		!strings.HasPrefix(proxyUrl, "https://") {
+		proxyUrl = fmt.Sprintf("%s://%s", defaultScheme, proxyUrl)
```

Comment:
So that mean now we are using the `http` scheme for `https` or `http` proxy is user didn't specify it explicit which sound good to me because `http` scheme can be used for both without any issue.